contributingGuides/LEFT_HAND_NAVIGATION.md

```diff
@@ -0,0 +1,44 @@
+## OVERVIEW
+
+The Left Hand Navigation (LHN) is designed to show different types of reports based on their status, user settings, and specific conditions. Each report type has unique visual indicators and sorting rules to help users quickly identify and prioritize their tasks.
+
+### Types of report displayed in the LHN
+
+The following outlines the expected behavior regarding which reports are displayed in the LHN:
+
+- The report currently being viewed by the user is highlighted as the active report in the LHN, making it easy for users to locate their focus point within the navigation.
+- If a report has unresolved issues, like an unapproved expense or outstanding violations, it will display a red dot next to it, indicating urgent action is required. These reports are displayed at the top of the LHN list and sorted alphabetically by report name for easy access.
```

Comment:
```suggestion
- If a report has unresolved issues, like an unapproved expense or outstanding violations, it will display a red dot next to it, indicating urgent action is required. These reports are displayed at the top of the LHN list (under pinned chats) and sorted alphabetically by report name for easy access.
```